### ISSUE TITLE
Overrride table name

### ISSUE THREAD

**Issue by damianh** [AllocZero/EfficientDynamoDb#106]
AWS's sdk allows overrding of the table name attribute: https://github.com/aws/aws-sdk-net/blob/475822dec5e87954b7a47ac65995714ae1f1b115/sdk/src/Services/DynamoDBv2/Custom/DataModel/Configs.cs#L119 . This is useful for single table design approachs and decoupling the table name (a deployment concern I will argue) from the code. Is the same possible with EfficientDynamoDb?

**Comment by lezzi** [comment]
Currently, it is not possible, but it sounds like a good candidate for the fluent API. I was actually considering adding it at some point but then it was forgotten:

```csharp
var items = await ddbContext.Query<EntityClass>()
    .WithKeyExpression(condition)
    .WithTableName("overriden_table")
    .ToListAsync();
```

**Comment by lezzi** [comment]
Should now be available in [0.9.1-alpha.0.17](https://www.nuget.org/packages/EfficientDynamoDb/0.9.1-alpha.0.17)
